opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -189,6 +189,30 @@ def observe(self, value: ValueT, labels: Dict[str, str]) -> None:
         """
 
 
+class SumObserver(Observer):
```

Comment:
I think you need to update the `ObserverT` TypeVar in this file

Reply:
Currently we have `ObserverT = TypeVar("ObserverT", bound=Observer)`. I believe as long as the class used is a subclass of "Observer" this definition should be accurate?

Reply:
Oh sorry you're right!